package/src/renderer/__tests__/Transform.spec.tsx

```diff
@@ -46,4 +46,21 @@ describe("Renderer", () => {
     );
     processResult(surface, "snapshots/transform/scale-origin.png");
   });
+  it("Use gradient has the unit for rotation", () => {
```

Comment:
Description in the form of:

`it("should render gradient transformed xx degrees to the right")`

It is much more easy to verify and make sure we

Reply:
👍🏻